source/apps/copy.txt

```diff
@@ -0,0 +1,200 @@
+.. _copy-app:
+
+===========
+Copy an App
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+You can make a copy of an existing App by reusing the App's
+configuration files and manually porting other information like Secrets.
+
+You might want to copy an application if:
+
+- You use feature branches for development. Use a unique copy of the
+  App for each feature branch to avoid conflicts.
+
+- You run tests against a working version of the App. You can copy an
+  App for each test run to to ensure a consistent start state.
+
+- You deploy the same app to clusters across regions using a :ref:`local
+  deployment model <local-deployment>`. You can copy an App to servce
+  multiple regions locally.
+
+Before You Begin
+----------------
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: cli
+
+      You will need the following to copy an App in the CLI:
+
+      - .. include:: /includes/prereqs/atlas-account.rst
+
+      - .. include:: /includes/prereqs/atlas-admin-api-key.rst
+
+      - .. include:: /includes/prereqs/app-services-cli.rst
+
+   .. tab::
+      :tabid: github
+
+      To copy an App based on configuration files in a GitHub
+      repository, you must enable :ref:`Automatic GitHub Deployment
+      <deploy-github>` for the App.
+
+Procedure
+---------
+
+.. include:: /includes/warning-github-deploy-dont-use-cli.rst
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: cli
+
+      .. procedure::
+
+         .. step:: Authenticate a MongoDB Atlas User
+
+            .. include:: /includes/cli-login.rst
+
+         .. step:: Create a New App
+
+            Create a new blank App. Choose the same name as the original
+            App and set the deployment model and region to be what you
```

Comment:
why must name be the same?

Reply:
good point - can copy the config to an app with a different name